doc/Support/FAQ.md

```diff
@@ -317,9 +317,13 @@ architecture then the following steps should be all that's needed:
 - and import it into your new server (`mysql -u root -p < librenms.sql`).
 - Copy the `rrd/` folder to the new server.
 - Copy the `config.php` file to the new server.
+- Check that no specific os have been added on includes/definitions
```

Comment:
I'm not sure about this one.  Maybe just a generic statement about modified files instead of a specific one about yaml...

Reply:
@murrant I updated the doc, thanks !